### PR TITLE
Fix Key Vault live tests for SKR

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

**Comment by heaths** [comment]
@schaabs @benbp @christothes sorry for the seeming noise on the permissions. I sorted them so it's easier to spot changes in the future, and added "release" to key permissions.

**Comment by heaths** [comment]
Also based on conversation with service team, adding "release" unconditionally to Key Vaults that may not support it yet in some clouds shouldn't pose an issue during provisioning.

**Comment by heaths** [comment]
Relates to #22750

**Review by benbp — APPROVED** [review]

File: sdk/keyvault/Azure.Security.KeyVault.Keys/tests/KeyClientLiveTests.SecureKeyRelease.cs [hunk 1]
```diff
@@ -3,18 +3,19 @@
...
 using Azure.Core.TestFramework;
+using Azure.Security.KeyVault.Tests;
 using NUnit.Framework;
 
 namespace Azure.Security.KeyVault.Keys.Tests
 {
     public partial class KeyClientLiveTests
     {
         [Test]
+        [PremiumOnly]
```
Comment (christothes) [inline sdk/keyvault/Azure.Security.KeyVault.Keys/tests/KeyClientLiveTests.SecureKeyRelease.cs:18]: Do we run test suites against both premium and non-premium SKUs?

**Review by christothes — APPROVED** [review]

File: sdk/keyvault/Azure.Security.KeyVault.Keys/tests/KeyClientLiveTests.SecureKeyRelease.cs (hunk 1 above)
Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Keys/tests/KeyClientLiveTests.SecureKeyRelease.cs:18]: Yes, in our weekly test runs.

**Pull request merged**